README.md

```diff
@@ -1,62 +1,129 @@
-# eigenda-blob-scraper
+# EigenLayer AVS OnChain Exporter
 
-A Python application that scrapes the Eigenda website for the latest blob data and exposes it via Prometheus metrics.
+EigenLayer AVS OnChain Exporter (EOE) is a tool designed to export on-chain data from various Active Validator Set (AVS) services within the EigenLayer ecosystem as Prometheus metrics.
 
-## Project Description
+## Features
 
-The application gets the blob data in a json form from an EigenDA public endpoint and transforms it into Prometheus metrics every `FETCH_INTERVAL` seconds. The metrics are then exposed via a Prometheus server at port 9600.
+### Supported AVSs and Prometheus metrics
 
-The application is deployed via Docker image to DockerHub. The image can be found here: https://hub.docker.com/repository/docker/nethermind/eigenda-blob-scraper.
+#### EigenDA
 
-## Environment Variables
+For EigenDA, Holeksy and Mainnet are supported and exposes the following metrics:
 
-The application uses the following environment variables:
+- `eoe_eigenda_exporter_latest_block{network="<network>"}`: Latest block number that the EigenDA exporter of the specific network has processed.
+- `eoe_eigenda_onchain_batches_total{network="<network>"}`: Total number of onchain batches that the EigenDA exporter of the specific network has processed. This is a counter that increments with each block and resets to 0 if the exporter is restarted.
+- `eoe_eigenda_onchain_batches{operator="<operator>", network="<network>", status="<status>"}`: Number of onchain batches missed by an operator in the specific network. For now the only status is `missed`.
+- `eoe_eigenda_onchain_quorum_status{operator="<operator>", network="<network>", quorum="<quorum>"}`: The status of the operator in the specific network and quorum. The value could be 1 if the operator is in quorum, 0 if the operator is not in quorum.
 
-| Variable | Description | Default Value |
-| --- | --- | --- |
-| API_URL | Endpoint to get the blob json data from | https://blobs-goerli.eigenda.xyz/api/trpc/blobs.getBlobs |
-| FETCH_INTERVAL | Fetch interval in seconds | 60 |
+> Due this exporter could be tracking operators that are already running before the exporter is deployed, the `eoe_eigenda_onchain_quorum_status` initial value could be configured by setting the `operators[i].eigenDAConfig.quorums[j]` to `true` if the operator is in quorum at exporter start.
```

Comment:
I'm not checking the initial status of the operator for each quorum so that I will delegate this declaration to the user. This is an improvement that could be done. Check the operator quorum status by checking the on-chain data

Reply:
Right, you only update the status upon receiving a new event in that regard. I see there could be a mismatch in the quorum status if you provide the value. It can be error-prone because it requires human intervention. Think if this is something we would like to have for this release or another dev cycle.

Reply:
I prefer to move it to the next dev cycle. We need to have this tool ASAP

Reply:
Please rephrase/rewrite/clarify that text

Reply:
Done https://github.com/NethermindEth/eigenlayer-onchain-exporter/pull/3/commits/bd0f68eba6de2601e562173c9200ce8a8e3128ad